Exec/RegTests/EB-ConvergingNozzle/README.md

```diff
@@ -0,0 +1 @@
+# Simple flow in a converging nozzle geometry
```

Comment:
Maybe specify subsonic here + how BCs are defined.

Mention how to check for the problem that led to this test being added?